roles/core/nic/defaults/main.yml

```diff
@@ -1 +1,2 @@
+---
 nic_nm_controlled: yes
```

Comment:
@oxedions I introduced this file with ea3c977b while I did not know yet you don't want to use defaults in the roles. Should we rework this?